packages/app/src/specs/generators/empty/EmptyGenerator.tsx

```diff
@@ -7,5 +7,10 @@ export const EmptyGenerator: SpecGenerator = {
   entry: EmptyGeneratorCardStepOne,
   matches: () => true,
   disabled: () => false,
+  show: (currentProject) => {
+    const specPattern = currentProject?.config.find((item) => item.field === 'specPattern')
```

Comment:
We could add a field in gql for `isDefaultSpecPattern` so we don't have t derive it on the frontend.

Reply:
I like this. Our array-based config lookups always feels unwieldy.